extensions/wikia/Chat2/js/controllers/controllers.js

```diff
@@ -78,7 +78,7 @@ var NodeChatSocketWrapper = $.createClass(Observable, {
 		});
 	},
 
-	onConnect: function (socket) {
+	onConnect: function (socket, transport) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "transport". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)